src/LayerTree/LayerTree.spec.jsx

```diff
@@ -61,14 +59,15 @@ describe('<LayerTree />', () => {
   });
 
   it('unmount removes listeners', () => {
-    const unByKeySpy = sinon.spy(olObservable, 'unByKey');
+    const unByKeySpy = jest.spyOn(olObservable, 'unByKey');
     const wrapper = TestUtil.mountComponent(LayerTree, {map});
     const olListenerKeys = wrapper.instance().olListenerKeys;
     wrapper.unmount();
-    expect(unByKeySpy.callCount).toBe(1);
-    expect(unByKeySpy.calledWith(olListenerKeys)).toBeTruthy();
+    expect(unByKeySpy).toHaveBeenCalled();
+    expect(unByKeySpy.mock.calls[0][0]).toBe(olListenerKeys);
```

Comment:
We might better use `.toHaveBeenCalledWith`  https://facebook.github.io/jest/docs/en/expect.html#tohavebeencalledwith